### ISSUE TITLE
"Fatally wounded" optional message.

### ISSUE THREAD

**Issue by MrLantis** [pwatson100/symbaroum#580]
I recommend making the "fatally wounded" message of the combat enhancement optional so that you as a GM can hide the fact that your players one-hit KOed your boss fight...

**Comment by bithir** [comment]
Should this setting also avoid printing pain threshold message?

**Comment by Pheromant** [comment]
> Should this setting also avoid printing pain threshold message?

Yes. This should be possible for all messages that display the result of an attack.

**Comment by Pheromant** [comment]
Another option could be to make this message „hidden“ so the GM can see it and make it „public“ once the „apply results“ button has been pushed.
